schema.graphql

```diff
@@ -102,3 +102,8 @@ type Account @entity {
   id: ID!
   fpmmPoolMemberships: [FpmmPoolMembership!] @derivedFrom(field: "funder")
 }
+
+type TokenRegistry @entity {
+  id: ID!
+  markets: [String!]
```

Comment:
I actually think you can use [ID!] instead of [String!] here. Also, instead of trying to figure out whether it's empty or null, why not use [ID!]!?